eth/bft/bft_handler.go

```diff
@@ -26,9 +26,9 @@ type Bfter struct {
 	broadcast        BroadcastFns
 
 	// Message Cache
-	knownVotes     *lru.ARCCache
-	knownSyncInfos *lru.ARCCache
-	knownTimeouts  *lru.ARCCache
+	knownVotes     *lru.Cache
```

Comment:
I saw this on lru document. Is this the reason we don't use ARC?
> ARC has been patented by IBM, so do not use it if that is problematic for your program.

Reply:
no, simple just follow other Known message cache lib. Also ARCCache doesn't have `ContainsOrAdd` function, that's why I check and use Simple lru.Cache.